packages/nimble-components/src/table/specs/table-column-width-hld.md

```diff
@@ -15,27 +15,45 @@ We need to provide users the means for changing the widths of individual columns
-    -   A sizing action to the left will ultimately stop having an effect when the left-most column reaches its minimum size.
-    -   A sizing action to the right that would ultimately result in the final right column reaching its minimum size (all columns still within current table width) would then begin to push columns out of the table width resulting in a horizontal scrollbar.
+-   If a user drags a divider between two columns to the right, then the column on the left will grow larger, and the column on the right will grow smaller by the same pixel amount. Sub-behaviors to this are: - If a shrinking column has reached its minimum pixel size or is not resizable, then the next column in the direction of the sizing action will be affected up to the final column in a given direction.
+-   A sizing action to the left will ultimately stop having an effect when the left-most column reaches its minimum size. - A sizing action to the right that would ultimately result in the final right column reaching its minimum size (all columns still within current table width) would _conditionally_ (see behavior descriptions below) begin to push columns out of the table viewport width resulting in a horizontal scrollbar on the table. - Within a single mouse interaction (i.e. drag-sizing without releasing mouse), if a cascade results in a column not adjacent to the divider being sized, then moving the mouse back in the opposite direction will "revert" the size made to the non-adjacent column.
+    ![Column resizing](spec-images/tableColumnResize.gif)
+
 -   Columns can be configured to not allow a user to interactively size them
     -   The implicit behavior present based on the behaviors described above, is that in a sizing action that cascades to a column configured to not be resized is that the column won't be resized towards a minimum size, and the cascade will effectively "skip" this column.
 
+#### Table Sizing
+
+-   Table resizing can result in _asymmetric_ column size state for mirrored actions such as a window maximize operation followed by a window restore down action. The following is a summary of expected behavior:
+    -   If the table currently has a horizontal scrollbar and the table is then sized to be wider, then as the table grows it will first remove excess space for the viewport prior to growing any columns. At this point, after the initial wider size has been completed, further reductions in window size will _not_ have a restorative state to the scrollbar prior to growing the window.
+    -   If the table currently has a horizontal scrollbar and the table is then sized to be smaller, the current proportional width of each column will be maintained.
+
+[Prototype branch](https://60e89457a987cf003efc0a5b-xtgmcptvna.chromatic.com/?path=/story/table-column-configuration--grouping).
```

Comment:
nit/fyi: stories from builds are deleted after 30 days, so if possible include a gif, but not a deal-breaker (wouldn't expect prototype branches to stick around forever anyway)

Reply:
I've now just changed the link to the working branch (not a prototype branch anyway). I've also included another gif to demonstrate proposed behavior.